notary-status.md

```diff
@@ -0,0 +1,183 @@
+
+As a result, we identified the following stages of the Notary v2 effort:
+
+1. Define Requirements
+1. Build Prototypes
+1. Validate prototypes - learning, refining requirements, iterating prototypes
+1. Author a Notary v2 Spec
+
+## February 2021
+
+Since December 2019, teams have been meeting regularly, as captured in the [Notary v2 notes][nv2-notes]. Requirements and a focused prototype have been completed, enabling conversations to upstream projects.
+
+Updates on the **Areas of Focus** and **Stages** are provided below.
+
+### Stage Progress (Feb 2021)
+
+1. [**Define Requirements**](#define-requirements-(2021-02))
+1. [**Build Prototypes**](#build-prototypes-(2021-02))
+1. [**Validate Prototypes**](#validate-prototypes-(2021-02))
+1. [*Author a Notary v2 Spec*](#author-a-notary-v2-spec-(2021-02))
+
+#### Define Requirements (2021-02)
+
+[Baseline requirements](https://github.com/notaryproject/requirements#requirements) have been defined, enabling focused prototyping. Additional requirements are being discussed as PRs:
+
+- [Add ephemeral clients as a core scenario #25](https://github.com/notaryproject/requirements/issues/25)
+- [Should tags be signed in addition to digests? #43](https://github.com/notaryproject/requirements/issues/43)
+
+#### Build Prototypes (2021-02)
+
+In September of 2020, a focused prototype for signing, integration with a registry and client validation were completed.
+
+The prototype focused on:
+
+- Signing a container image with a prototype of an [nv2 client](https://github.com/notaryproject/nv2).
+- Pushing to an instance of the [CNCF Distribution][cncf-distribution] based registry, with exploratory [prototype enhancements](https://github.com/notaryproject/distribution/pull/2).
+- Discovery of signatures, based on a [verification by reference](https://github.com/notaryproject/requirements/blob/main/verification-by-reference.md) design.
+- Validation of a signature with the `nv2` client, using the public key pair used to sign the artifact.
+
+Details of the September 2020 Prototype:
+
+- [Video of prototype-1](https://youtu.be/ciNW19F_T_8?t=367)  
+- [Steps to reproduce the prototype-1 nv2 client](https://github.com/notaryproject/nv2/tree/prototype-1/docs/nv2)
+
+The prototypes have enabled participants and subject matter experts to identify additional areas of focus, such as [policy management capabilities](https://github.com/notaryproject/requirements/issues/42). Policy management will be added as an area of focus.
```

Comment:
This feels like a huge expansion in scope IMO.

I think there's probably a difference between prototyping to ensure policy management can coexist with the signature formats/retrieval/etc. and making it a core part of the notary project.

Which one is implied here?

Reply:
it’s a good check. 
There have been a number of great questions that have come up, such as:
- How many keys should be used to verify an artifact
- Should the CN of the cert match the registry the artifact is currently in?
  - one in the ACME Rockets environment, the wabbit networks key can be used to verify the content, but the registry would match the ACME Rockets CN. 
  - issues around key revocation, such as delays in air-gapped environments

The Policy Management area is a bucket for these topics as we don’t believe they should be core to key validation, rather options in policy management. To your scope question, I don’t expect this would replace OPA/GateKeeper, rather integrate with these type of policy management solutions.

Reply:
Update pushed. Please re-open if I didn't capture the question.